corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap3/main.js

```diff
@@ -409,15 +409,28 @@ hqDefine('hqwebapp/js/bootstrap3/main', [
             var alertCookie = 'alerts_maintenance';
             var closedAlerts = $.cookie(alertCookie) ? JSON.parse($.cookie(alertCookie)) : [];
 
+            var viewedDomainAlertsCookie = 'viewed_domain_alerts';
```

Comment:
Used a separate cookie for domain alerts instead of extending the one available just above, since this needs to be reset on login.